dist/vivus.js

```diff
@@ -719,6 +734,40 @@ Vivus.prototype.mapping = function() {
   }
 };
 
+/**
+ * Prublic method re-evaluate line length on non-scaling
+ * path elements.
+ */
+Vivus.prototype.recalc = function () {
+  if (this.mustRecalcScale) {
+    return;
+  }
+  this.mustRecalcScale = requestAnimFrame(function () {
+    this.recalcThrottle();
+  }.bind(this));
+}
+
+/**
+ * Private method to re-evaluate line length on non-scaling
+ * path elements. Then call for a trace to update the SVG. 
+ */
+Vivus.prototype.recalcThrottle = function () {
```

Comment:
This isn't actually a throttle or doing any throttling. It's also technically not private because it's on the prototype?

```suggestion
Vivus.prototype.performLineRecalc = function () {
```